src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/form_div_layout.html.twig

```diff
@@ -340,9 +340,13 @@
 {%- endblock form_errors -%}
 
 {%- block form_rest -%}
+  {% import '@PrestaShop/Admin/macros.html.twig' as ps %}
```

Comment:
@matks not sure if this thing is evil or not but if I try to import it on the top of the file I get the following message:

![screen3](https://user-images.githubusercontent.com/17051250/58029417-2ec24200-7b25-11e9-8633-9348b20d9536.png)
so right now it stays inside the block and thats the thing I am worried about. Do you know if its good or bad?

Reply:
Yes because you're already in a block context, if you import a template, it's trying to extends it.
Keep it in the wanted block.